static/i18n/en/errors.yml

```diff
@@ -1,97 +1,97 @@
+  description: Needlessly tried to update the device microcontroller. Contact Ledger Support if there's a problem with your device.
 LedgerAPIError:
-  title: 'Ledger API error. Try again. (HTTP {{status}})'
-  description: ''
+  title: 'Sorry, try again (API HTTP {{status}})'
+  description: Interacting with Ledger's API server went wrong. Please retry.
 LedgerAPIErrorWithMessage:
-  title: '{{message}}'
-  description: ''
+  title: 'Oops, {{message}})'
+  description: Please retry or contact Ledger Support.
 LedgerAPINotAvailable:
-  title: 'Ledger API not available for {{currencyName}}.'
-  description: ''
+  title: Sorry, {{currencyName}} services unavailable
+  description: Please retry or contact Ledger Support.
 ManagerAPIsFail:
-  title: Services are unavailable. Please try again.
-  description: ''
+  title: Oops, Manager services unavailable.
+  description: Please check the network status. #link to status.ledger.fr ?
 ManagerAppAlreadyInstalled:
-  title: App is already installed
-  description: ''
+   title: Oops, that's already installed. # include {{currencyName}}
+   description: Check your device to see which apps are already installed.
 ManagerAppRelyOnBTC:
-  title: You must install Bitcoin application first
-  description: ''
+  title: Bitcoin app required
+  description: Install the Bitcoin app before installing this app. # include {{currencyName}}
 ManagerDeviceLocked:
-  title: Device is locked
-  description: ''
+  title: Please unlock your device
+  description: Your device was locked. Please unlock it.
 ManagerNotEnoughSpace:
-  title: Not enough storage on device
-  description: Uninstall some apps and try again.
+  title: Sorry, insufficient device storage
+  description: Uninstall some apps to increase available storage and try again.
 ManagerUnexpectedError:
-  title: Unexpected error occurred ({{msg}})
+  title: That's unexpected ({{msg}}) #(Manager: {{msg}})
   description: Please try again.
 ManagerUninstallBTCDep:
-  title: You must uninstall other altcoins first
-  description: ''
+  title: Sorry, Bitcoin is required # include {{currencyName}}
+  description: First uninstall apps that depend on Bitcoin.
 NetworkDown:
-  title: 'Your internet connection seems down.'
-  description: ''
+  title: Oops, internet seems down
+  description: Please check your internet connection.
 NoAddressesFound:
-  title: 'No accounts were found.'
-  description: ''
+  title: Sorry, no accounts found
+  description:  # Missing context
 NotEnoughBalance:
-  title: 'Insufficient funds to proceed.'
-  description: ''
+  title: Oops, not enough balance
+  description: Make sure the account to debit has sufficient balance
 RangeError:
-  title: '{{message}}'
-  description: ''
+  title: '{{message}}' # Missing context
+  description:
 TimeoutError:
-  title: 'The request timed out.'
-  description: ''
+  title: Oops, a time out occurred
+  description: It took too long for the server to respond. #Retry / check status?
```

Comment:
yeah, usually we will offer a Retry in place. it depends where (ofr instance for Sync it's the sync button etc. etcc)